### PR TITLE
release v3.5.0

### PR DESCRIPTION
**Pull request by tzha-plaid**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Changelog
+
+## 3.5.0
```
Comment (clayallsopp) [inline CHANGELOG.md:3]: @tzha-plaid bump `package.json` too?

Comment (tzha-plaid) [reply CHANGELOG.md:3]: good catch!

**Comment by deepdil-sp** [comment]
facing issue on this release

**Comment by tzha-plaid** [comment]
what issue are you facing?

**Pull request merged**